src/workloads/execution/Densify.yml

```diff
@@ -0,0 +1,194 @@
+SchemaVersion: 2018-07-01
```

Comment:
Worth finding out if there is any kind of best practice to number of measurements per file.

Reply:
Asked on slack, but looking at the other workloads it seems like the general idea is to have the workload as the logical unit of whatever you're testing.

Reply:
Depending on the answer to one of my questions later down in the workload, we might have to separate these existing operations out into even more test cases. In that case, it seems like the numeric testing and the date-type testing is fairly distinct - how do you feel about separating these tests into two workloads, something like `DensifyNumeric` and `DensifyDate`? This would make the workloads each more digestible, and also allow you to be more specific about document fields for each workload and the indexes you need to add for a covered $sort.

Reply:
Would splitting up the workloads make it more difficult to compare the metrics between numeric and date workloads? While they're distinct in the code, I think it would be valuable to be able to compare the performance between both types, since I wouldn't expect them to be too different and a huge discrepancy would be something to look into.

Reply:
That's a good point! My gut reaction is that apples-to-apples comparison across workloads would be fairly simple as long as we keep the relative document size and count the same. All in all it seems like a matter of preference, so as long as we don't find the length of this workload growing too unwieldy, I'm fine with either option.

Reply:
Hmm after your last change this is growing pretty unwieldy; I think over 700 lines of a workload is a little too much for reading comprehension in my eyes. Could you split this into multiple workloads?

Reply:
After splitting up the workloads, one thing I'm worried about is for some of the common configuration getting out of sync. The metrics across different files won't be able to be usefully compared to each other unless the same documents are loaded in for both, and the queries are equivalent in both files. Some things like the step sizes for small steps and large steps and the batch size are factored out into YAML anchors, which could pretty easily get out of sync. Wondering if that should be a concern, or if the workloads will be changing so infrequently as to not really matter.

Reply:
Ended up splitting up into four workloads, one for each unit.